.github/workflows/publish.yaml

```diff
@@ -0,0 +1,91 @@
+name: Publish Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        description: 'Version of the latest release (e.g. v0.42.0)'
+        required: false
+        default: ''
+  repository_dispatch:
+    types: [ocm-cli-release]
+
+jobs:
+
+  push-to-aur:
+    name: Update Arch Linux User Repository
+    runs-on: ubuntu-latest
+    steps:
+    - name: Ensure proper version
+      run: |
+        if [ -n "${{ github.event.inputs.version }}" ]; then
+          echo "RELEASE_VERSION=$(echo ${{ github.event.inputs.version }} | tr -d ['v'])" >> $GITHUB_ENV
+          cat $GITHUB_ENV
```

Comment:
@frewilhelm at that point in time the env variable should be pretty empty and only contain whatever a GitHub actions contains. Since we have not enriched it with anything from the Github secrets section, e.g. a GitHub token, the env should not contain any sensitive value.

Reply:
> [...] should not contain any sensitive value

such statements make me anxious :D

Reply:
removed it